orttraining/orttraining/training_api/checkpoint.cc

```diff
@@ -551,9 +553,19 @@ Status OrtLoadInternal(const PathString& checkpoint_path,
     LoadTensorProtoFromFile(tensor_file_full_path, tensor_protos, "[params]");
 
     for (const auto& tensor_proto : tensor_protos) {
```

Comment:
```suggestion
    for (auto& tensor_proto : tensor_protos) {
```
would need to be non-const reference in order to actually move from it

Reply:
So in this case If I used const it will copy ?

Reply:
yep. some more info here:
https://www.nextptr.com/tutorial/ta1211389378/beware-of-using-stdmove-on-a-const-lvalue